### ISSUE TITLE
unicode under windows

### ISSUE THREAD

**Issue by Zimmi48** [Zimmi48/bugzilla-test#2686]
Note: the issue was created automatically with bugzilla2github tool

# Bugzilla Bug ID: 2686
Date: 2012-01-25 10:54:42 +0100
From: @_pirbo
To:   
CC:   @_pirbo

Duplicates:   #2675
Depends on:   #1379
Last updated: 2012-01-25 11:02:41 +0100

## Bugzilla Comment ID: 4718
Date: 2012-01-25 10:54:42 +0100
From: @_pirbo

-Compiling a buffer from the menu affects the source file! in particular, utf-8
characters are actually replaced by their code.
In particular, opening the file utf8.v (which is lacking from the windows
package, btw), and compiling it from the menu makes utf8.v bigger while
importing the newly created utf8.vo (with the command Require Import utf8.) then
replaces the forall's by \u2200 instead of the pretty symbol.
Clearly the utf-8 encoding is lost at some point in the compilation.

## Bugzilla Comment ID: 4723
Date: 2012-01-25 11:02:41 +0100
From: @_pirbo

*** This bug has been marked as a duplicate of bug #2675 ***
